app/presenters/citizen_readiness/link_presenter.rb

```diff
@@ -0,0 +1,30 @@
+module CitizenReadiness
+  class LinkPresenter
+    include Tracking
+
+    attr_reader :base_path
+
+    def initialize(base_path, index)
+      @base_path = base_path
+      @index = index #used in link tracking
+    end
+
+    def title
+      I18n.t("campaign.links.#{slug}.title")
+    end
+
+    def description
+      I18n.t("campaign.links.#{slug}.description")
+    end
+
+    def link
+      base_path
+    end
+
+  private
+
+    def slug
+      link.delete("/")
```

Comment:
This won't support multiple `/` in a base_path. Is that an issue?

Reply:
It's just to drop the leading `/` so should be fine for now I think?

Reply:
Cool :)